cmd/soci/commands/create.go

```diff
@@ -29,10 +29,11 @@ import (
 )
 
 const (
-	buildToolIdentifier = "AWS SOCI CLI v0.1"
-	spanSizeFlag        = "span-size"
-	minLayerSizeFlag    = "min-layer-size"
-	optimizationFlag    = "optimizations"
+	buildToolIdentifier    = "AWS SOCI CLI v0.1"
+	spanSizeFlag           = "span-size"
+	minLayerSizeFlag       = "min-layer-size"
+	forceXAttrsEnabledFlag = "force-xattrs-enabled"
```

Comment:
Can we just name this option "disable-xattrs" and default to true?

Reply:
urfave/cli doesn't allow bool flags to default to true, but I agree that it's a bit of a wordy flag name. Unsure of an alternative variable name that would not confuse users. Maybe something like `--skip-xattr-check` or something? Though it's not much better either...

Reply:
I believe you can do it with [BoolTFlag](https://github.com/urfave/cli/blob/v1.22.14/flag_bool_t.go#L10) :)

Reply:
Oh that's pretty neat. Just pushed changes.